### DISCUSSION TITLE
Minimum hw specs to enable video streaming?

### DISCUSSION THREAD

**Discussion by fpaddict** [dgreif/ring#1076] [Q&A]
I have a Raspberry Pi Model B Plus Rev 1.2 and when I try to stream video from any of my Ring cameras, the CPU spikes to 100% and the video never comes through; the Home app says "No Response. This camera is not responding". Snapshots seem to work fine.

The log shows:

![image](https://user-images.githubusercontent.com/116017241/197179765-a597ce8e-9d70-4b13-9390-e41170f426de.png)

I was wondering if there are minimum specs for the hardware to enable streaming?

To try to help, I've unbridged the cameras and running the plug in as a child bridge.

**Comment by dgreif** [comment]
I'd say the RPi 3 is the lowest I would try to run it on. Those logs look like a successful connection, so it's possible your old RPI just isn't keeping up with transcoding
